app/javascript/iiif.js

```diff
@@ -1,18 +1,13 @@
-import Mirador from '@columbia-libraries/mirador/dist/es/src';
-import miradorDownloadPlugins from '@columbia-libraries/mirador/dist/es/src/culPlugins/mirador-downloaddialog';
-import canvasRelatedLinksPlugin from '@columbia-libraries/mirador/dist/es/src/culPlugins/mirador-canvasRelatedLinks'
-import citationSidebar from '@columbia-libraries/mirador/dist/es/src/culPlugins/mirador-citations';
-import hintingSidebar from '@columbia-libraries/mirador/dist/es/src/culPlugins/mirador-hinting-sidebar';
-import videoJSPlugin from '@columbia-libraries/mirador/dist/es/src/culPlugins/mirador-videojs';
-import viewerNavigation from '@columbia-libraries/mirador/dist/es/src/culPlugins/mirador-pageIconViewerNavigation';
-import viewXmlPlugin from '@columbia-libraries/mirador/dist/es/src/culPlugins/mirador-viewXml';
-import collectionFoldersPlugin from '@columbia-libraries/mirador/dist/es/src/culPlugins/mirador-selectCollectionFolders';
+import Mirador from '@columbia-libraries/mirador';
 
-const flattenPluginConfigs = (...plugins) => plugins.reduce(
-  (acc, curr) => {
-    return acc.concat([...curr])
-  }, []
-);
+const culMiradorPlugins = [...Mirador.culPlugins.downloadDialogPlugin]
+  .concat([...Mirador.culPlugins.viewXmlPlugin])
+  .concat([...Mirador.culPlugins.citationsSidebarPlugin])
+  .concat([...Mirador.culPlugins.videojsPlugin])
+  .concat([...Mirador.culPlugins.canvasRelatedLinksPlugin])
+  .concat([...Mirador.culPlugins.hintingSideBar])
+  .concat([...Mirador.culPlugins.viewerNavigation])
+  .concat([...Mirador.culPlugins.nativeObjectViewerPlugin]);
```

Comment:
```suggestion
const culMiradorPlugins = [
  ...Mirador.culPlugins.downloadDialogPlugin,
  ...Mirador.culPlugins.viewXmlPlugin,
  ...Mirador.culPlugins.citationsSidebarPlugin,
  ...Mirador.culPlugins.videojsPlugin,
  ...Mirador.culPlugins.canvasRelatedLinksPlugin,
  ...Mirador.culPlugins.hintingSideBar,
  ...Mirador.culPlugins.viewerNavigation,
  ...Mirador.culPlugins.nativeObjectViewerPlugin
];
```
Would this work instead?

Reply:
These are reasonable questions, but (for right now) I think they should move over to the Mirador fork unless you are observing a problem when the code runs or flagging a potential bug in a change in this PR - this is basically copied from finding aids, which is in large measure copied from https://github.com/cul/mirador/blob/cul-v4.0.0-alpha.10-rebase/__tests__/integration/mirador/cul.html

Reply:
Makes sense!